bitnami/redis-cluster/templates/networkpolicy.yaml

```diff
@@ -25,16 +25,16 @@ spec:
         protocol: UDP
     # Allow outbound connections to other cluster pods
     - ports:
-        - port: {{ .Values.redis.port }}
-        - port: {{ .Values.redis.busPort }}
+        - port: {{ .Values.redis.ports.redis }}
+        - port: {{ .Values.redis.ports.bus }}
```

Comment:
You can introduce backwards compatibility avoiding introducing a major version. Check how @migruiz4 did it for MariaDB using `coalesce`:
- https://github.com/bitnami/charts/pull/7563

Reply:
We are adding the missing labels to the volumeClaimTemplate, changing the labels in the statefulsets are usually problematic on upgrades, so if are going to maintain a major version bump I would not support the old way.